diesel/src/query_dsl/belonging_to_dsl.rs

```diff
@@ -1,5 +1,46 @@
+/// Loads record(s) by a directional association with (an)other record(s).
+///
+/// # Example
+///
+/// ```rust
+/// // You need to have the table definitions from `infer_schema!` or `table!` in scope to
+/// // derive Identifiable.
+/// # #[macro_use] extern crate diesel;
+/// # #[macro_use] extern crate diesel_codegen;
+/// # include!("../doctest_setup.rs");
+/// # use schema::{posts, users};
+///
+/// # #[derive(Identifiable, Queryable)]
+/// # pub struct User {
+/// #     id: i32,
+/// #     name: String,
+/// # }
+///
+/// # #[derive(Debug, PartialEq)]
+/// # #[derive(Identifiable, Queryable, Associations)]
+/// # #[belongs_to(User)]
+/// # pub struct Post {
+/// #     id: i32,
+/// #     user_id: i32,
+/// #     title: String,
+/// # }
+///
+/// # fn main() {
+/// # let connection = establish_connection();
+/// # use users::dsl::*;
+/// # let user = users.find(2).get_result::<User>(&connection).unwrap();
+/// let posts = Post::belonging_to(&user)
+///     .load::<Post>(&connection);
+///
+/// assert_eq!(posts,
+///     Ok(vec![Post { id: 3, user_id: 2, title: "My first post too".to_owned() }])
+/// );
+/// # }
+/// ```
 pub trait BelongingToDsl<T> {
+    ///
     type Output;
 
+    /// Get the record(s) belonging to record(s) `other`
```

Comment:
Do you think that this implies we're loading the data instead of returning a query that needs to be executed?